components/admin_console/admin_definition.jsx

```diff
@@ -2502,25 +2510,31 @@ export default {
                                     display_name: t('atmos/camo'),
                                     display_name_default: 'atmos/camo',
                                 },
+                                {
+                                    value: 'local',
+                                    display_name: t('local'),
+                                    display_name_default: 'local',
+                                },
                             ],
+                            isDisabled: needsUtils.stateValueFalse('ImageProxySettings.Enable'),
                         },
                         {
                             type: Constants.SettingsTypes.TYPE_TEXT,
-                            key: 'ServiceSettings.ImageProxyURL',
+                            key: 'ImageProxySettings.RemoteImageProxyURL',
                             label: t('admin.image.proxyURL'),
                             label_default: 'Image Proxy URL:',
                             help_text: t('admin.image.proxyURLDescription'),
                             help_text_default: 'URL of your image proxy server.',
-                            isDisabled: needsUtils.stateValueFalse('ServiceSettings.ImageProxyType'),
+                            isDisabled: needsUtils.stateValueFalse('ImageProxySettings.Enable'),
```

Comment:
Similarly, we could also disable them if the image proxy type is "local" as well